test/gurka/test_exclude_unpaved_smoothness.cc

```diff
@@ -179,11 +179,16 @@ TEST(Standalone, SmoothnessAccess) {
     const DirectedEdge* edge_2 = nullptr;
 
     std::tie(edge_id_1, edge_1, edge_id_2, edge_2) = findEdge(*graph_reader, map.nodes, "AB", "B");
-    // no access due to smoothness = impassable and are therefore tossed.
+    // ped only access due to smoothness = impassable.
     // edge_1 = AB
     // edge_2 = BA
-    EXPECT_EQ(edge_1, nullptr);
-    EXPECT_EQ(edge_2, nullptr);
+    EXPECT_NE(edge_1->forwardaccess(), baldr::kPedestrianAccess);
```

Comment:
Shouldn't this be `EXPECT_EQ`? What other access does it have?

Reply:
yes.  let me fix that.  Probably has wheelchair and pedestrian

Reply:
@chrstnbwnkl thank you for seeing that issue.  Test was all wrong.  Good to go now.